Client/Frontend/Browser/Favorites/FavoritesViewController.swift

```diff
@@ -742,7 +516,226 @@ extension FavoritesViewController {
         } else {
             // User doesn't want to see the recent searches option again
             Preferences.Search.shouldShowRecentSearchesOptIn.value = false
-            collectionView.reloadData()
         }
     }
 }
+
+// MARK: - Diffable data source + NSFetchedResultsControllerDelegate
```

Comment:
What does this mean, `MARK`?

Reply:
@bsclifton It's an Xcode-specific syntax that adds text to the file navigator and minimap:
<img width="145" alt="image" src="https://user-images.githubusercontent.com/529104/128743218-057e8019-14a6-4180-bc08-1b77797dca20.png">
<img width="376" alt="image" src="https://user-images.githubusercontent.com/529104/128743235-572989b5-2832-49e7-86cf-de17baf80537.png">

Reply:
It is a visual thing, splits sections of code in XCode navigator, for example 

<img width="969" alt="xcode_mark_lines_example" src="https://user-images.githubusercontent.com/6950387/128743344-89d37a31-b4cf-4493-b5ca-c4ef827e5e57.png">

Reply:
Also marks minimap my favourite :)

<img width="351" alt="Screen Shot 2021-08-09 at 1 04 04 PM" src="https://user-images.githubusercontent.com/6643505/128745472-df1a7129-f929-41e1-b5ba-b17dc38a2e55.png">

This is similar to what we do in objective-c code `#pragma mark ` in brave-core.